### DISCUSSION TITLE
TagoIO Integrated(via TTS) | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1592] [Q&A]
# TagoIO Integrated(via TTS) | Seeed Studio Wiki

SenseCAP_T1000_tracker_and_TagoIO_Integrated

https://wiki.seeedstudio.com/SenseCAP_T1000_tracker_TagoIO_TTS/

**Comment by ito2897** [comment]
Hello.

I selected a location in "Data from: Your device -- location"

but this location is not displayed.

What should I do?
